src/api/documents/collection-ref.ts

```diff
@@ -26,43 +26,62 @@ import { DocumentRef } from './document-ref';
  */
 export class CollectionRef<T extends { [key: string]: any }> {
   private documentClient: DocumentServiceClient;
-  private _collection: Collection;
+  public readonly name: string;
+  public readonly parent?: DocumentRef<any>;
 
   constructor(
     documentClient: DocumentServiceClient,
     name: string,
-    parentKey?: Key
+    parent?: DocumentRef<any>,
   ) {
     this.documentClient = documentClient;
+    this.name = name;
+    this.parent = parent;
+  }
 
-    const collection = new Collection();
-    collection.setName(name);
-
-    if (parentKey) {
-      collection.setParent(parentKey);
-    }
-
-    this._collection = collection;
+  /**
+   * Return a collection group reference from this collection
```

Comment:
I wonder if a comment with an example could make this more clear, something like:
```
Return a reference to a subcollection within the documents of this collection.

Useful when querying subcollection documents across all/many parent documents. E.g. Querying landmarks from multiple cities.
```
Don't need to do this now, just a thought (we should make it consistent between SDKs too)

Reply:
done